### PR TITLE
mgr/dashboard: additional fixes to block pages

### PR DESCRIPTION
**Pull request by dillaman**

### CODE REVIEW

**Comment by rjfd** [comment]
@dillaman this is great! Thanks for fixing the iSCSI part of the dashboard.

**Comment by LenzGr** [comment]
Indeed - thanks a lot for your help!

**Comment by dillaman** [comment]
no worries

File: src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi/iscsi.component.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { Component } from '@angular/core';
 
+import { CellTemplate } from '../../../shared/enum/cell-template.enum';
 import { CephShortVersionPipe } from '../../../shared/pipes/ceph-short-version.pipe';
 import { DimlessBinaryPipe } from '../../../shared/pipes/dimless-binary.pipe';
```
Comment (ricardoasmarques) [inline src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi/iscsi.component.ts]: Now that we are no longer using the `dimlessBinaryPipe` in this file, we can remove this import and the corresponding line on the class constructor (line 25).

**Pull request merged**